hack/run-ci-e2e-test.sh

```diff
@@ -86,12 +86,25 @@ fi
 # https://steps.svc.ci.openshift.org/help/ci-operator#release
 OPERATOR_IMAGE=${OPERATOR_IMAGE:-${IMAGE_FORMAT//\/stable:\$\{component\}//stable:windows-machine-config-operator-test}}
 
-# Setup and run the operator
-if ! run_WMCO $OSDK; then
-  # Try to get the WMCO logs if possible
-  get_WMCO_logs
-  cleanup_WMCO $OSDK
-  exit 1
+# OO_INDEX points to the WMCO bundle index image created through https://docs.ci.openshift.org/docs/how-tos/testing-operator-sdk-operators/ workflow in CI.
+# Setup and run the operator only if $OO_INDEX is not set in CI, and the operator deployment is not available.
+OO_INDEX=${OO_INDEX:-}
+
+# SKIP_OPERATOR_RUN_CLEANUP will skip operator deployment and cleanup if set to true
+SKIP_OPERATOR_RUN_CLEANUP=${SKIP_OPERATOR_RUN_CLEANUP:-"false"}
+
+if [ -z "$OO_INDEX" ] && ! oc rollout status deployment windows-machine-config-operator -n openshift-windows-machine-config-operator; then
```

Comment:
What's the reason for checking `oc rollout status deployment windows-machine-config-operator -n openshift-windows-machine-config-operator`?

Reply:
to make sure we only skip deployment when the operator is actually running, added this as an additional guard to skip our manual deployment

Reply:
This is so that we can keep calling `olm.sh run` even when the operator is running?

Reply:
not really, olm.sh calls run_wmco() independently, but my main reason was to keep this guard until we have confidence in this process, we can remove it once everything is in good shape.

Reply:
please add a TODO comment for this ^